### PR TITLE
Updates ref docs for `*ByKey` functions now point to Contract Keys ref docs

### PR DESCRIPTION
**Pull request by akrmn**
the explanations in "Reference: Contract Keys" seem correct and generally clearer to me. Keeping a single source of truth will also prevent conflicting docs in the future.

### CODE REVIEW

File: docs/2.6.0/docs/daml/reference/updates.rst
Comment (garyverhaegen-da) [inline docs/2.6.0/docs/daml/reference/updates.rst:142]:
```suggestion
************
```

File: docs/2.7.0/docs/daml/reference/updates.rst
Comment (garyverhaegen-da) [inline docs/2.7.0/docs/daml/reference/updates.rst:142]:
```suggestion
************
```

**Review by garyverhaegen-da — APPROVED** [review]
LGTM, thanks!

File: docs/2.6.0/docs/daml/reference/updates.rst
Comment (akrmn) [reply docs/2.6.0/docs/daml/reference/updates.rst]: thanks!

**Comment by garyverhaegen-da** [comment]
@akrmn Is there more to do here or can I merge this?

**Comment by akrmn** [comment]
@garyverhaegen-da, I'd like to wait for some reviews from @digital-asset/daml-language

**Review by remyhaemmerle-da — APPROVED** [review]
LGTM

**Pull request merged**
